### DISCUSSION TITLE
Unable to get `ExpressAuth` working in monorepo

### DISCUSSION THREAD

**Discussion by haroon-schmrkow** [nextauthjs/next-auth#11280] [Help]
I am unable to make the basic example from Next-Auth docs work in my monorepo setup with Express.js.

I am using `"@auth/express": "^0.5.6"` and the error I get is

```
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/.../gno/apps-monorepo/node_modules/@auth/express/index.js from /Users/.../gno/apps-monorepo/apps/api/src/server.ts not supported.
api:dev: Instead change the require of index.js in /Users/.../gno/apps-monorepo/apps/api/src/server.ts to a dynamic import() which is available in all CommonJS modules.
```

I am importing like following: `import { ExpressAuth } from "@auth/express";` which is how I import everything else...

In my `package.json` I don't have the `type` property set to anything.

Do you have any ideas what could be the issue here?

**Comment by lucasrothman** [comment]
@haroon-schmrkow Did you ever figure it out? Struggling with the same issue I think, reposted here https://github.com/nextauthjs/next-auth/discussions/11417

**Comment by hillac** [comment]
I answered here https://github.com/nextauthjs/next-auth/discussions/11417#discussioncomment-10195832
